src/main/java/com/microsoft/cse/helium/app/Constants.java

```diff
@@ -16,6 +16,10 @@
   public static final String WEB_INSTANCE_ROLE = "WEBSITE_ROLE_INSTANCE_ID";
   public static final Integer MAX_DEGREE_PARALLELISM = 2;
 
+  public static final String AUTH_TYPE = "AUTH_TYPE";
+  public static final String KEY_VAULT_NAME = "KeyVaultName";
+  public static final String USE_MSI = "MSI";
+  public static final String USE_CLI = "CLI";
```

Comment:
Also need to support VS

Reply:
@bartr, do we support VS for Java?

Reply:
How does the VS credentials work with JAVA? Haven't seen it before

Reply:
Java is not supported in Visual Studio, only in VS code. Does any of the other languages work with the VS Code and is there a visual studio credentials system that works? Ref: https://visualstudio.microsoft.com/vs/features/web/languages/

Reply:
Per team agreement, add the argument but return E_NOIMPL